ZLocation/ZLocation.psm1

```diff
@@ -65,8 +65,14 @@ Update-ZLocation $pwd
 #
 # Tab completion.
 #
-if (Test-Path Function:\TabExpansion) {
-    Rename-Item Function:\TabExpansion PreZTabExpansion
+Register-ArgumentCompleter -CommandName Set-ZLocation -ParameterName match -ScriptBlock {
```

Comment:
cool, that's great! Let's just add fences against the case where `Register-ArgumentCompleter` is not available, i.e. old PS or no PSReadLine. Here an example https://github.com/PowerShell/platyPS/blob/9c2da487bb8ad9dd837d6f85cc733f51009ea6f2/src/platyPS/platyPS.psm1#L2991